pkg/cmd/server/bootstrappolicy/constants.go

```diff
@@ -142,3 +147,19 @@ const (
 	BuildStrategySourceRoleBindingName          = BuildStrategySourceRoleName + "-binding"
 	BuildStrategyJenkinsPipelineRoleBindingName = BuildStrategyJenkinsPipelineRoleName + "-binding"
 )
+
+// Resources and Subresources
+const (
```

Comment:
Put in a synthetic.go file so we know these are used for virtual SARs

Reply:
I can add a comment if *really* needed, I see no reason to add a new file

Reply:
Add a comment then.

Reply:
Ok, let me ask, why do we care for saying these constants are synthetic ?
Is there some fundamental property we need to highlight ?
To me a constant is a constant is a constant ...

Reply:
They are special because we made them up or in some way gave them meaning.  `get pods` is obvious because we know what a pod is.  This is not true for the synthetic checks because they do not line up with an API resource.